research-software-directory/RSD-as-a-service
# Styled AppHeader

# Unified and styled App header

Fixes #307

<img width="1671" alt="image" src="https://user-images.githubusercontent.com/4195550/173667671-a263f9d3-13f6-464b-869f-3608dea9bc61.png">

Changes proposed in this pull request:

- [x] New app header styles and same as the landing page

This PR contains some code cleaning with a few files deleted for simplification

How to test:

* `make start`, and check that the app header is equal and behaves as it was before. 

PR Checklist:
